skimage/filters/_fft_based.py

```diff
@@ -77,6 +85,13 @@ def butterworth(
     channel_axis : int, optional
         If there is a channel dimension, provide the index here. If None
         (default) then all axes are assumed to be spatial dimensions.
+    squared_butterworth : bool, optional
+        When True, the square of a Butterworth filter is used. See notes below
+        for more details.
+    npad : int, optional
+        Pad each edge of image the by `npad` pixels using `numpy.pad`'s
+        ``mode='edge'`` extension. Try increasing `npad` if boundary artifacts
```

Comment:
I'd say this info (_"Try increasing `npad` if boundary artifacts are apparent."_) should be in the Notes section.

Reply:
seems reasonable. I have moved it now